docs/Contributors.md

```diff
@@ -2,24 +2,6 @@
 
 They helped [develop the app](https://github.com/lwouis/alt-tab-macos/graphs/contributors):
 
-* [adamnemecek](https://github.com/adamnemecek)
```

Comment:
This happens if you run `scripts/update_contributors.sh` without setting `$GITHUB_TOKEN`. You can simply remove [this line](https://github.com/lwouis/alt-tab-macos/blob/master/scripts/update_contributors.sh#L16) before running the script (and change the `\` on the previous line with `|`) to make the github call without a token. See the comment above the line for more explanations on why the token is needed in the first place

Reply:
I assumed if I built a PR, it would have a chance to run on Travis once. I removed the contributors so I could have a successful RUN there once. I didn't see that info, I will try that out, thanks.

Reply:
> I assumed if I built a PR, it would have a chance to run on Travis once.

Yes, PRs on this repo trigger a CI build. You can see the builds generated by this PR here: https://github.com/lwouis/alt-tab-macos/pull/1094/checks. You can then access travis itself by clicking the links, and seeing the logs. It's all open to anyone to peruse.

> I removed the contributors so I could have a successful RUN there once

It's a tricky situation for PRs. There are scripts like `scripts/update_contributors.sh` that exist to make sure that contributors are credited; that adding them to the list is not forgotten. That runs on PRs as well. It can be annoying because unrelated to the PR. On the other hand, if we disable those for PRs, then someone could add a new library and forget to credit it, or share commits from someone else and forget to credit them. Same for localization scripts. They make sure people are aware and do the right things when introducing new strings in the app which will need to be localized (easy to forget, especially for first-time contributors)

My flow here is to run the `scripts/update_contributors.sh` locally with the token line removed, then the contributors are up to date and I can move on.

If you have any idea to improve this, please feel free to share

Reply:
That's very honorable in principle, but just doesn't meet my expectation when I create a PR that then never build the binary. Because I don't want to put my name in there unless I could build a successful build on Travis with the setup.

Reply:
How would you suggest we improve this?

Reply:
I would suggest doing it only on the `master` branch.